library/general/src/lib/installation/proposal_client.rb

```diff
@@ -132,6 +137,11 @@ def run
     #     The caller will format each item as a HTML list item (`<li>`). The
     #     proposal can contain hyperlinks with IDs listed in the list `links`.
     #
+    #   * **`"label_proposal"`** [Array<String>, nil]
```

Comment:
well, we can define, that with "cramped_label_mode" it do not contain it. I just do not like to return hash with possible bunch of keys. If we define that if this key is passed it have to return "raw_proposal" without any richtext tags, it will reduce list of possible return states.